resources/views/content/scripts.twig

```diff
@@ -1,20 +1,17 @@
 <script type="text/javascript">
-    window.flSearchSubmitEvent = document.createEvent('Event');
-    window.flSearchSubmitEvent.initEvent('fl-search-submit', false, true);
-    document.addEventListener('fl-search-submit', function (e) {
-        e.preventDefault();
+    function flSearchSubmitEventHandler(event, element) {
```

Comment:
Rename `element` to `form` or `formElement` or even `searchFormElement` (you can decide), so it is clear that we are dealing with a form.